bazel/envoy_test.bzl

```diff
@@ -326,3 +344,65 @@ def envoy_sh_test(
             tags = tags + ["nocoverage"],
             **kargs
         )
+
+def _envoy_fuzzing_launcher_impl(ctx):
```

Comment:
What part of this overlaps with rules-fuzzing? (Is it the same besides the `need_launcher`? Is this something that will be replaced by https://github.com/googleinterns/bazel-rules-fuzzing/pull/56/files?